src/main/java/org/cryptomator/cryptofs/health/shortened/ShortenedNamesCheck.java

```diff
@@ -103,6 +115,36 @@ void checkShortenedName(Path dir) throws IOException {
 			}
 		}
 
+
+		/**
+		 * Checks if the string stored inside the name file is base64url encoded
+		 *
+		 * <em>visible for testing</em>
+		 *
+		 * @return {@link NameEncoding} indicating if it is valid, invalid or affected by https://github.com/cryptomator/cryptofs/issues/121
+		 */
+		NameEncoding isValidEncodedName(String toAnalyse) {
```

Comment:
Since this is no longer a question of base64 encoding or not, I'd strip the word "encoding" from this method (and return type). Rather use something like `ContentRating validateC9sFileContent(String content)`.

Reply:
Fixed in db83548